go/libraries/events/event_flush_test.go

```diff
@@ -0,0 +1,87 @@
+package events
+
+import (
+	"context"
+	"path/filepath"
+	"testing"
+
+	eventsapi "github.com/liquidata-inc/dolt/go/gen/proto/dolt/services/eventsapi_v1alpha1"
+	"github.com/liquidata-inc/dolt/go/libraries/utils/filesys"
+	"github.com/stretchr/testify/assert"
+	"google.golang.org/grpc"
+)
+
+var globalEvents []*eventsapi.ClientEvent
```

Comment:
using a global will be problematic if 2 of these tests run concurrently.  I'd make this a member of the TestClient